src/kv/Rpc.cc

```
@@ -1,9 +1,42 @@
 {
+namespace
+{
+bool isConnValid(const std::shared_ptr<KvConnClient> & conn_client, size_t rpc_timeout)
+{
+    auto state = conn_client->channel->GetState(false);
+    if (state == GRPC_CHANNEL_READY)
+        return true;
+
+    auto deadline = std::chrono::system_clock::now() + std::chrono::seconds(rpc_timeout);
+    if (conn_client->channel->WaitForConnected(deadline))
+        return true;
```

Clang (14.0.6)

[error] redundant boolean literal in conditional return statement

(readability-simplify-boolean-expr,-warnings-as-errors)